.changeset/happy-fans-share.md

```diff
@@ -0,0 +1,5 @@
+---
+'@contentful/f36-autocomplete': patch
+---
+
+Enable controlling the dorpdown menu open and close state after select with new property "closeAfterSelect"
```

Comment:
I don't know if that is relevant but minor typo "dorpdown" :)

Reply:
🧐 thanks